### ISSUE TITLE
Allow setting custom context properties for applicationinsights.exceptions.enable()

### ISSUE THREAD

**Issue by letmaik** [Microsoft/ApplicationInsights-Python#103]
Currently there doesn't seem to be an easy way to use `applicationinsights.exceptions.enable()` while also sending context properties for those unhandled exceptions. Looking at the code it seems that one has to do something like the following currently (not tested):

```py
from applicationinsights import channel, TelemetryClient
from applicationinsights.exceptions import enable

channel = channel.TelemetryChannel()
client = TelemetryClient("my-key", channel)
client.context.properties["foo"] = "bar"
enable("my-key", telemetry_channel=channel) # must be kw arg
```

I think https://github.com/Microsoft/ApplicationInsights-Python/issues/56 would help for this case as well so that the user doesn't have to deal with TelemetryChannel objects.

Given bug https://github.com/Microsoft/ApplicationInsights-Python/pull/102 I think that the above code currently wouldn't work anyway since the `track_exception` call that happens in the background doesn't get `properties` which triggers the bug where the context properties are ignored as well.

**Comment by letmaik** [comment]
Correction, my example code should probably be:
```py
channel = channel.TelemetryChannel()
channel.context.properties["foo"] = "bar"
enable("my-key", telemetry_channel=channel) # must be kw arg
```

**Comment by letmaik** [comment]
As there's a fix for #102 I'll close this as it's not a big deal in terms of difficulty, more of a documentation issue.
